docs/src/modules/components/Link.tsx

```diff
@@ -71,7 +71,7 @@ const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(function Link(props,
   } = props;
 
   const router = useRouter();
-  const pathname = typeof href === 'string' ? href : href.pathname;
+  const pathname = typeof href === 'string' ? href : href?.pathname;
```

Comment:
@siriwatknp What was the issue with this one?

Reply:
to prevent the error in this case:

```js
<Link
  href={someValue} // someValue === undefined
/>
```
It is very hard to know which Link cause the error but it is easier to find which link does not have `href` (in the devtool)

Reply:
yes, of course!

Reply:
will fix them all in a separate PR.